common/dynamicconfig/constants.go

```diff
@@ -1178,6 +1178,13 @@ const (
 	// Allowed filters: ShardID
 	ReplicationTaskProcessorErrorRetryMaxAttempts
 
+	// MaxBufferedReplicationTasks is the max number of workflows for which we store in memory information
+	// Keyname: history.MaxWorkflowCacheSize
```

Comment:
Is this cache used for Replication tasks only or is it used for other things? If it's replication only, may I suggest we add Replication into the key-name?

Reply:
sorry thats a copy paste error

This is the workflow rate limit cache size, fixed the name - sorry about the confusion